src/Parser/OrientationParser.js

```diff
@@ -0,0 +1,119 @@
+        elements[1] = east.y; elements[5] = north.y; elements[9] = up.y;
+        elements[2] = east.z; elements[6] = north.z; elements[10] = up.z;
+        return target.setFromRotationMatrix(matrix);
+    };
+})();
+
+/**
+ *
+ * @typedef Attitude
+ * @type {Object}
+ *
+ * @property {Number} omega - angle in degrees
+ * @property {Number} phi - angle in degrees
+ * @property {Number} kappa - angle in degrees
+ * @property {Number} roll - angle in degrees
+ * @property {Number} pitch - angle in degrees
+ * @property {Number} heading - angle in degrees
+ */
+
+
+const ENUQuat = new THREE.Quaternion();
+
+export default {
+
+    /**
+     * @function quaternionFromAttitude
+     * @param {Attitude} attitude - [Attitude]{@link module:OrientationParser~Attitude}
+     * with properties: (omega, phi, kappa), (roll, pitch, heading) or none.
+     * Note that convergence of the meridians is not taken into account.
+     * @param {THREE.Quaternion} target Quaternion to set
+     * @returns {THREE.Quaternion} target
+     */
+    quaternionFromAttitude(attitude, target = new THREE.Quaternion()) {
+        if ((attitude.roll !== undefined) || (attitude.pitch !== undefined) || (attitude.heading !== undefined)) {
+            return quaternionFromRollPitchHeading(attitude.roll, attitude.pitch, attitude.heading, target);
+        }
+        if ((attitude.omega !== undefined) || (attitude.phi !== undefined) || (attitude.kappa !== undefined)) {
+            return quaternionFromOmegaPhiKappa(attitude.omega, attitude.phi, attitude.kappa, target);
+        }
+        return target.set(0, 0, 0, 1);
+    },
+
+    /**
+     * @function quaternionFromAttitudeGlobe
+     * @param {Attitude} attitude - [Attitude]{@link module:OrientationParser~Attitude}
+     * with properties: (omega, phi, kappa), (roll, pitch, heading) or none.
+     * @param {Coordinates} coordinate position on the globe
+     * @param {THREE.Quaternion} target Quaternion to set
+     * @returns {THREE.Quaternion} target
+     */
+    quaternionFromAttitudeGlobe(attitude, coordinate, target = new THREE.Quaternion()) {
+        quaternionENUFromGeodesicNormal(coordinate.geodesicNormal, ENUQuat);
+        this.quaternionFromAttitude(attitude, target);
+        target.premultiply(ENUQuat);
+        return target;
+    },
+
+    /** Parse rotation information (roll pitch heading or omega phi kappa),
+     * and create a ThreeJs quaternion representing a rotation.
+     *
+     * @param {Attitude} attitude - [Attitude]{@link module:OrientationParser~Attitude}
+     * @param {Coordinates} coordinate position the oject (used to apply another rotation on Globe CRS)
+     * @param {Boolean} needsENUFromGeodesicNormal should be true on globe CRS.
+     * If true, we will apply another rotation : The rotation use to create ENU local space at coordinate parameter position.
+     * @returns {THREE.Quaternion} rotation
+     */
+    parse(attitude, coordinate, needsENUFromGeodesicNormal) {
```

Comment:
The geojson parsing  is done in the `GeoJsonParser`. All the responsibility for parsing should be in this file.
It's easier to parse the file once and it will give simpler examples